redwood-treehouse-host/src/iosMain/kotlin/app/cash/redwood/treehouse/TreehouseUIView.kt

```diff
@@ -23,11 +23,18 @@ import platform.CoreGraphics.CGRectZero
 import platform.UIKit.UITraitCollection
 import platform.UIKit.UIView
 
-@ObjCName("TreehouseUIKitView", exact = true)
-public class TreehouseUIKitView private constructor(
+@Deprecated(
+  "Renamed to `TreehouseUIView` for consistency with other `TreehouseView` implementations.",
+  ReplaceWith("TreehouseUIView", "app.cash.redwood.treehouse.TreehouseUIView"),
+)
+// `TreehouseUIKitView` should be the name in Objective C, but `@ObjCName` cannot be applied to a typealias.
```

Comment:
This makes the change API breaking. This doesn't affect the guest-host bridge contract though, so I'm assuming that such a breaking change is okay.

Reply:
Yep